src/main/java/org/opentripplanner/middleware/docs/ReadMeEnvSchemaValuesUpdater.java

```diff
@@ -60,7 +60,7 @@ public static String generateEnvSchemaValuesContent() throws IOException {
                 .anyMatch(propertyName::equals))
                 ? "Required"
                 : "Optional";
-            String defaultValue = getJSONPropertyFieldAsString(property, "default");
+            String examples = getJSONPropertyFieldAsString(property, "examples");
```

Comment:
Just curious, would it make sense to resolve `examples` to `String[]` or `List<String>`?